src/app/home/home.component.html

```diff
@@ -62,8 +62,8 @@
   </div>
 </div>
 <!-- Create Space modal -->
-<modal #createSpace title="Create a new collaboration space">
-  <modal-content>
-    <space-dialog></space-dialog>
+<modal #createSpace class="chromeless-modal"  >
```

Comment:
agreed ...the modal paradigm we have still needs work i.m.o ...  chromeless dialogs typically defer look/feel/control to the contained component making things very simple ... the current paradigm splits this functionality into the host dialog/component making it complicated to control look and feel... a modal should ideally follow the single responsibility principle of just making the component modal and offer lifecycle hooks